index.html

```diff
@@ -10,6 +10,56 @@
 </head>
 
 <body>
+    <nav>
+        <div class="flex justify-between align-center mb-6 mx-4 mt-4 flex-wrap gap-x-1 top-navbar">
+            <div class="flex align center justify-between gap-x-4">
+                <a href="https://unsplash.com/"> <img src="./assets/unsplash-logo.webp" alt="logo" width="55"> </a>
+                <form class="relative text-input-with-icon">
+                    <input class="text-input text-input-with-icon search-input" placeholder=" Search" type="text"
```

Comment:
This is confusing. We have the `text-input-with-icon` on both the form and icon? I would expect it to be only on the input.

Please refactor so that it's meant to be added on the input.